packages/environment-ember-loose/-private/intrinsics/action.d.ts

```diff
@@ -12,36 +12,34 @@ export type ActionResult<T, Args extends ActionNamedArgs<T>> = undefined extends
 
 export type ActionKeyword = DirectInvokable<{
   <Ret, Args extends ActionNamedArgs<Ret>, Params extends unknown[]>(
-    args: Args,
-    f: (...rest: Params) => Ret
+    f: (...rest: Params) => Ret,
```

Comment:
The pattern does begin to feel less weird once you've internalized it, and it would be a bit annoying to extract everywhere, so maybe that's an argument *not* to extract like I suggested above.